deps/patchelf.mk

```diff
@@ -9,7 +9,7 @@ $(SRCDIR)/srccache/patchelf-$(PATCHELF_VER)/source-extracted: $(SRCDIR)/srccache
 	touch -c $(SRCDIR)/srccache/patchelf-$(PATCHELF_VER)/configure # old target
 	echo 1 > $@
 
-$(BUILDDIR)/patchelf-$(PATCHELF_VER)/build-configured: $(SRCDIR)/srccache/patchelf-$(PATCHELF_VER)/source-extracted | $(LIBCXX_DEPENDENCY)
+$(BUILDDIR)/patchelf-$(PATCHELF_VER)/build-configured: $(SRCDIR)/srccache/patchelf-$(PATCHELF_VER)/source-extracted $(LIBCXX_DEPENDENCY)
```

Comment:
we shouldn't need to rebuild patchelf anytime LIBCXX_DEPENDENCY is rebuilt (`|` means that it only depends on existence, not timestamp)